### PR TITLE
Combine masking logger calls

### PR DESCRIPTION
**Pull request by rjgildea**
* Combine up to six consecutive `logger.info()` calls into a single call
* Change to debug-level log messages

### CODE REVIEW

**Comment by graeme-winter** [comment]
👍

graylog2 also says thank you

**Comment by rjgildea** [comment]
Tested with:
```
$ dials.generate_mask $(dials.data get -q centroid_test_data)/*0001.cbf d_min=2 d_max=10 resolution_range=4,4.5 untrusted.rectangle=100,200,100,200 untrusted.circle=100,100,10 border=10 polygon=100,100,100,200,200,200,200,100 ice_rings=True -vv
```

File: util/masking.py [hunk 1]
```diff
@@ -271,20 +270,24 @@ def generate(self, imageset):
...
-                        logger.info(" panel = %d" % region.panel)
-                        logger.info(" xc = %d" % xc)
-                        logger.info(" yc = %d" % yc)
-                        logger.info(" radius = %d" % radius)
+                        logger.debug(
+                            "Generating circle mask:\n"
+                            + f" panel = {region.panel}\n"
+                            + f" xc = {xc}\n"
+                            + f" yc = {yc}\n"
+                            + f" radius = {radius}"
```
Comment (ndevenish) [inline util/masking.py:278]: Assuming the split lines are deliberate, I think the `+` are redundant here e.g. 
```suggestion
                            f" panel = {region.panel}\n"
                            f" xc = {xc}\n"
                            f" yc = {yc}\n"
                            f" radius = {radius}"
```

File: util/masking.py [hunk 2]
```diff
@@ -271,20 +270,24 @@ def generate(self, imageset):
...
-                        logger.info(" y0 = %d" % y0)
-                        logger.info(" x1 = %d" % x1)
-                        logger.info(" y1 = %d" % y1)
+                        logger.debug(
+                            "Generating rectangle mask:\n"
+                            + f" panel = {region.panel}\n"
+                            + f" x0 = {x0}\n"
+                            + f" y0 = {y0}\n"
+                            + f" x1 = {x1}\n"
+                            + f" y1 = {y1}"
```
Comment (ndevenish) [inline util/masking.py:289]:
```suggestion
                            f" panel = {region.panel}\n"
                            f" x0 = {x0}\n"
                            f" y0 = {y0}\n"
                            f" x1 = {x1}\n"
                            f" y1 = {y1}"
```

**Comment by ndevenish** [comment]
I've argued myself in circles over whether the concatenation makes sense so am going agnostic

**Pull request merged**